personal_assistant/pa_note/views.py

```diff
@@ -0,0 +1,50 @@
+
+from django.shortcuts import render, get_object_or_404, redirect
+from .models import Note
+from .forms import NoteForm
+from django.views import View
+
+
+def main(request):
+    return render(request, 'pa_core/home.html')
+
+
+def note(request):
+    notes = Note.objects.all()
+    return render(request, 'pa_note/note_list.html', {'notes': notes})
+
+
+class NoteCreateView(View):
```

Comment:
Приємно бачити, що використано стільки класів для обробки маршрутів проте в контексті застосунка зрозуміло, що всі вони відносяться до нотаток тож можна скоротити назви класів і не використовувати імʼя сунтості як префікс:
```suggestion
class CreationView(View):
```